src/cli/commands/bin.js

```diff
@@ -19,6 +19,6 @@ export function run(
     config.registries[RegistryYarn.registry].folder,
     '.bin',
   );
-  console.log(binFolder);
+  reporter.log(binFolder);
```

Comment:
This will output an ansi code at the beginning of the line that resets the cursor to the start making it unsuitable for consumption via another tool.